src/app/contextBroker/contextBroker.cpp

```diff
@@ -621,8 +622,9 @@ PaArgument paArgs[] =
   { "*",      ENT,       ENT_COMPS_V2,         ENT_COMPS_WORD,       badVerbGetPostOnly         }, \
                                                                                                    \
   { "GET",    IENT,      IENT_COMPS_V2,        IENT_COMPS_WORD,      getEntity                  }, \
-  { "POST",   IENTA,     IENT_COMPS_V2,        IENT_COMPS_WORD,      postAttributes             }, \
-  { "*",      IENT,      IENT_COMPS_V2,        IENT_COMPS_WORD,      badVerbGetPostOnly         }, \
+  { "DELETE", IENT,      IENT_COMPS_V2,        IENT_COMPS_WORD,      deleteEntity               }, \
+  { "POST",   IENT,      IENT_COMPS_V2,        IENT_COMPS_WORD,      postAttributes             }, \
+  { "*",      IENT,      IENT_COMPS_V2,        IENT_COMPS_WORD,      badVerbGetPostDeleteOnly   }, \
```

Comment:
We have a slight mismatch here in service-routine-names ...
- getEntity
- deleteEntity
- postAttributes

I believe we should pick one unique name, either 'Entity' or 'Attributes', so, either rename *postAttributes* to *postEntity*, or rename *getEntity* and *deleteEntity* to *getAttributes* and *deleteAttributes*.

Time to vote: which name is better: 'Entity' or 'Attributes' ?